src/components/Modal/Modal.vue

```diff
@@ -367,6 +379,15 @@ export default {
 			type: String,
 			default: 'body',
 		},
+
+		/**
+		 * Pass in true if you want the modal 'close' button to be displayed
+		 * within the modal boundaries.
+		 */
+		closeButtonContained: {
+			type: Boolean,
+			default: false,
```

Comment:
```suggestion
			default: true,
```
and mark the PR as breaking.